en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-260-to-300.md

```diff
@@ -37,314 +320,1598 @@ Follow the instructions below to move all the existing API Manager configuration
 
-1.  Download WSO2 API Manager Analytics 2.6.0 from [here](http://wso2.com/api-management/) .
-2.  Create a new database for the new statistics database and configure it in the `<APIM_ANALYTICS_2.6.0_HOME>/conf/worker/deployment.yaml` as follows:
-    In this example shows to configure MySQL database configurations.
+        ```
+        [database.config]
+        type = "mysql"
+        url = "jdbc:mysql://localhost:3306/config_db"
+        username = "username"
+        password = "password"
 
-    ``` java
-        - name: APIM_ANALYTICS_DB
-              description: "The datasource used for APIM statistics aggregated data."
-              jndiConfig:
-                name: jdbc/APIM_ANALYTICS_DB
-              definition:
-                type: RDBMS
-                configuration:
-                  jdbcUrl: 'jdbc:mysql://localhost:3306/spDatabase?autoReconnect=true'
-                  username: username
-                  password: password
-                  driverClassName: com.mysql.jdbc.Driver
-                  maxPoolSize: 50
-                  idleTimeout: 60000
-                  connectionTestQuery: SELECT 1
-                  validationTimeout: 30000
-                  isAutoCommit: false
-    ```
+        ```
 
-3.  Copy the relevant JDBC driver to the `<APIM_ANALYTICS_2.6.0_HOME>/lib` folder.
-4.  Start the new API-M Analytics server.
+4.  Update `<API-M_3.0.0_HOME>/repository/resources/conf/default.json` file by pointing to the WSO2UM_DB.
 
-    ``` java
-            cd <APIM_ANALYTICS_2.6.0_HOME>/bin
-            sh worker.sh
+    ```
+    "realm_manager": {
+        "data_source": "WSO2USER_DB",
+        "properties": {
+        "isCascadeDeleteEnabled": true,
+        "initializeNewClaimManager": true
+        }
+    }
     ```
 
-    When the WSO2 API-M Analytics 2.6.0 server starts, the new statistics tables will get generated in the database that you configured in **step 1.2.1 - (2.)** above.
+5.  Copy the relevant JDBC driver to the `<API-M_3.0.0_HOME>/repository/components/lib` folder.
 
-##### Step 1.2.2 - Configure WSO2 API-M 2.6.0 for Analytics
+    !!! info
+        In API-M 3.0.0, you do not need to configure the registry configurations as you did in the `<CURRENT_API-M_HOME>/repository/conf/registry.xml` file and the user database configurations as you did in in the `<CURRENT_API-M_HOME>/repository/conf/user-mgt.xml` file, as those configurations have been handled internally.
 
-Follow the instructions below to configure WSO2 API Manager for the WSO2 API-M Analytics migration in order to migrate the statistics related data.
+6.  Move all your Synapse configurations to API-M 3.0.0 pack.
+    -   Move your Synapse super tenant configurations.
+        Copy the contents in the `<CURRENT_API-M_HOME>/repository/deployment/server/synapse-configs/default` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/deployment/server/synapse-configs/default` directory with the copied contents.
```

Comment:
We need to add a note with the synapse files that should not be replaced. (eg - main.xml, cors_request_handler_.xml)

Reply:
added